vendor/bat-native-ledger/src/bat/ledger/internal/endpoint/payment/payment_util.cc

```diff
@@ -19,16 +21,23 @@ std::string GetServerUrl(const std::string& path) {
   DCHECK(!path.empty());
 
   std::string url;
-  switch (ledger::_environment) {
-    case type::Environment::DEVELOPMENT:
-      url = kDevelopment;
-      break;
-    case type::Environment::STAGING:
-      url = kStaging;
-      break;
-    case type::Environment::PRODUCTION:
-      url = kProduction;
-      break;
+  const base::CommandLine& command_line =
```

Comment:
Is this command line switch required? If so, then we'll need to put the command line parsing into `rewards_service_impl.cc` and then provide access to this value from the browser-side, either by:

- Setting a global `ledger::x` variable, or
- Providing it in response to a `GetStringOption` call.

Reply:
resolved: https://github.com/brave/brave-core/pull/7557/files#diff-b3817ac19a1c6ad3db2832fc878795ccc8a6f5fc0ae281cdf70d85a3a0a3d324R1537